microraiden/microraiden/close_all_channels.py

```diff
@@ -44,18 +49,9 @@
     default=None,
     help='Ethereum address of the channel manager contract'
 )
-def main(private_key, state_file, channel_manager_address):
+def main(private_key, private_key_password_file, state_file, channel_manager_address):
+    private_key = utils.get_private_key(private_key, private_key_password_file)
     if private_key is None:
```

Comment:
Can you make `--private-key` click option mandatory? It doesn't really make sense to start the proxy without privkey and click will display a nice error message if the option is missing.

Reply:
Done.  `click.option` has a `required` keyword, although options usually are meant to be... optional. But should work.